__tests__/endpoints.test.js

```diff
@@ -129,6 +129,50 @@ describe("'/api/articles' endpoint", () => {
             })
         })
     })
+    describe("GET relevant articles with filtered query on topic", () => {
+        test("GET /api/articles?topic=cats returns 200, on single match topic", () => {
```

Comment:
You don't need to specify the precise query in the test description.
Something like : GET /api/articles takes a topic query and returns articles on that topic